app/views/kiosk/index.html.erb

```diff
@@ -3,4 +3,29 @@
      data-kiosk-type="<%= @kiosk_type %>"
      data-slides='<%= raw(JSON.dump @slides) %>'
      data-kiosk-url="<%= kiosk_url id: @kiosk_type %>"
```

Comment:
the login link is actually being displayed from the main layout, I put it in a partial https://github.com/osulp/kiosks/blob/master/app/views/layouts/_user_menu.html.erb for the main application layout